File: image_submit.py
from clarifai.rest import ClarifaiApp
from clarifai.rest import Image as ClImage

app = ClarifaiApp(api_key='c4406989d7624ad0aa336616ea3e4ada')

model = app.models.get('Bugz')
image = ClImage(url='https://upload.wikimedia.org/wikipedia/commons/thumb/f/fa/Theraphosa_blondi_MHNT.jpg/220px-Theraphosa_blondi_MHNT.jpg')
model.predict([image])
print model.predict_by_url(url='https://upload.wikimedia.org/wikipedia/commons/thumb/f/fa/Theraphosa_blondi_MHNT.jpg/220px-Theraphosa_blondi_MHNT.jpg')

File: image_submit.js
//var i = 0;

//function hello() {
//  i = i + 1;
//  console.log("hello" + i);

//}
import { fileinput.value } from './image_upload.js';
const Clarifai = require('clarifai');

const app = new Clarifai.App({
  apiKey: 'c4406989d7624ad0aa336616ea3e4ada'
});

app.models.predict(Clarifai.GENERAL_MODEL, fileinput.value).then(
  function(response) {
    console.log(response);
  }
  function(err) {

  }
);
